solidity/contracts/test/BridgeStub.sol

```diff
@@ -0,0 +1,33 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.8.4;
+
+import "../bridge/BitcoinTx.sol";
+import "../bridge/Bridge.sol";
+
+contract BridgeStub is Bridge {
+    constructor(
+        address _bank,
+        address _relay,
+        address _treasury,
+        uint256 _txProofDifficultyFactor
+    ) Bridge(_bank, _relay, _treasury, _txProofDifficultyFactor) {}
+
+    function setMainUtxo(bytes20 walletPubKeyHash, BitcoinTx.UTXO calldata utxo)
+        external
+    {
+        mainUtxos[walletPubKeyHash] = keccak256(
+            abi.encodePacked(
+                utxo.txHash,
+                utxo.txOutputIndex,
+                utxo.txOutputValue
+            )
+        );
+    }
+
+    function setWallet(bytes20 walletPubKeyHash, Wallet calldata wallet)
+        external
+    {
+        wallets[walletPubKeyHash] = wallet;
+    }
```

Comment:
We can make our lives easier in future by passing `address _walletRegistry` to `Bridge` constructor and adding a function to the `Bridge` contract:

```
function onNewWalletCreated(bytes20 walletPubKeyHash, Wallet calldata wallet) external {
  require(msg.sender == walletRegistry, "Caller is not the wallet registry");
   wallets[walletPubKeyHash] = wallet;
}
```

The signature of this function will probably change, given that the hash will be probably computed in the `Bridge`, but at least we'll not have to touch the `Stub`. 

Leaving it up to you if it makes sense to do it in this PR or leave for later.

Reply:
I left a TODO since I think it's a topic strictly related to integration with the `WalletRegistry`. TODO added as part of https://github.com/keep-network/tbtc-v2/pull/128/commits/deef0f3a97fe4ede455bfb6a0da60e9a419c8799